src/WebJobs.Script/Binding/FunctionBinding.cs

```diff
@@ -134,5 +121,13 @@ protected string Resolve(string name)
 
             return _config.HostConfig.NameResolver.ResolveWholeString(name);
         }
+
+        internal static void AddStorageAccountAttribute(Collection<CustomAttributeBuilder> attributes, string connection)
```

Comment:
Should we introduce a `StorageFunctionBinding` type that storage bindings would derive from, and where we'd expose this method instead of having it in `FunctionBinding`? My concern is that when we start applying this to the other bindings (e.g. Event Hubs, Service Bus, Easy Tables, etc.), we'll end up with a bunch of `AddFooAccountAttribute` methods here. It would be nice if the core attribute was a more generic `ConnectionAttribute`, or something along those lines.